### PR TITLE
updated signup and signin modals

### PR DESCRIPTION
**Pull request by 1010varun**
Screenshot from 2023-02-17 23-08-04
Screenshot from 2023-02-17 23-07-57

@narayan954 please review and merge my pr

### CODE REVIEW

**Comment by 1010varun** [comment]
@narayan954, please review and merge my pr

**Comment by 1010varun** [comment]
@narayan954 please review my pr.

**Comment by narayan954** [comment]
> @narayan954 please review my pr.

thanks for the reminder, looking into it!

File: package.json [hunk 1]
```diff
@@ -17,10 +17,12 @@
     "@mui/icons-material": "^5.11.9",
     "@mui/material": "^5.11.9",
     "@mui/styles": "^5.11.9",
+    "bootstrap": "^5.2.3",
```
Comment (narayan954) [inline package.json:20]: bootstrap wasn't used so need to remove it

File: src/App.css [hunk 2]
```diff
@@ -1,41 +1,26 @@
...
-}
 
-@keyframes logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.file {
```
Comment (narayan954) [inline src/App.css:3]: app.css isn't needed at all. these two style blocks can be moved to index.css and remove app.css

File: src/App.jsx [hunk 3]
```diff
@@ -311,39 +317,36 @@ function App() {
...
-                variant="contained"
-                color="primary"
-                sx={buttonStyle}
-              >
+              <div className="file-input">
+                <input type="file" id="file" className="file" onChange={handleChange}
+                accept="image/*"/>
+                <label htmlFor="file">Select Profile Picture</label>
+              </div>
+              <AnimatedButton type="submit" variant="contained" color="primary">
```
Comment (narayan954) [inline src/App.jsx:349]: you forgot the style of animated button.

**Review by narayan954 — COMMENTED** [review]
you did great and it looks nice, but I'd like to make some amendments and fix a minor mistake as in review.

**Review by narayan954 — APPROVED** [review]
looks good to me

**Pull request merged**